src/internal/testutil/db.go

```diff
@@ -95,3 +95,7 @@ func GenerateEphemeralDBName(t testing.TB) string {
 	// 	now.Hour(), now.Minute(), now.Second(),
 	// 	rand.Uint32())
 }
+
+func SetCleanup(c bool) {
```

Comment:
This probably isn't a good idea to expose.  This isn't safe to call concurrently, a lot of tests are run in parallel.

I think `cleanup` is intended to be used during development and manually set as a constant.